lib/go/events/set_play_retired.go

```diff
@@ -0,0 +1,49 @@
+package events
+
+import (
+	"fmt"
+	"github.com/onflow/cadence"
+	jsoncdc "github.com/onflow/cadence/encoding/json"
+)
+
+var (
+	EventPlayRetiredFromSet = "TopShot.PlayRetiredFromSet"
+)
+
+type SetPlayRetiredEvent interface {
```

Comment:
Out of scope for this PR, but I'm not sure why these needed to be interfaces. Seems like we could have just used a struct type, e.g.

```golang
type SetPlayRetiredEvent struct {
    rawEvent cadence.Event
}

func (evt SetPlayRetiredEvent) SetID() uint32 {
	return uint32(evt.Fields[0].(cadence.UInt32))
}

etc...
```

Reply:
Or actually better:

```
type SetPlayRetiredEvent struct {
    SetID uint32
    PlayID uint32
    NumMoments uint32
}
```

And then done all the mapping in the `Decode` method.